AzureAuth.lutconfig

```diff
@@ -0,0 +1,6 @@
+<LUTConfig Version="1.0">
+  <Repository />
+  <ParallelBuilds>true</ParallelBuilds>
+  <ParallelTestRuns>true</ParallelTestRuns>
+  <TestCaseTimeout>30000</TestCaseTimeout>

```

Comment:
What does this timeout imply?

Reply:
I think this is just the time in milliseconds (so, 30 seconds) before a test case is considered timed out. Of course I can't find **any** documentation for this feature, so... that's just a guess.

Reply:
This is my understanding as well, except I changed it to 30 sec. It defaulted to 180... which is ridiculous for a unit test.

Reply:
What does test case timing out mean? This is just for my own understanding.

Reply:
It takes longer than 30 seconds to run.